src/Core/Services/Implementations/SendService.cs

```diff
@@ -201,5 +186,28 @@ public string HashPassword(string password)
         {
             return _passwordHasher.HashPassword(new User(), password);
         }
+
+        private async Task ValidateUserCanSaveAsync(Guid? userId)
+        {
+            if (!userId.HasValue || _currentContext.Organizations?.FirstOrDefault() == null)
```

Comment:
I feel like using Any() or Count()/Length reads more clear what you are doing here. Maybe something like:
```suggestion
            if (!userId.HasValue || !_currentContext.Organizations?.Any() ?? true)
```

Reply:
+1 on that

Reply:
sounds good. I didn't like the !Any, but I can be overruled ;)